atlasdb-client/src/main/java/com/palantir/atlasdb/table/description/render/StreamStoreRenderer.java

```diff
@@ -321,27 +326,56 @@ private void createTempFile() {
                 } line("}");
             }
 
-            private void loadSingleBlockToOutputStream() {
+            private void loadBlocksToOutputStream() {
                 line("@Override");
-                line("protected void loadSingleBlockToOutputStream(Transaction t, ", StreamId, " streamId, long blockId, OutputStream os) {"); {
-                    line(StreamValueRow, " row = ", StreamValueRow, ".of(streamId, blockId);");
-                    line("try {"); {
-                        line("os.write(getBlock(t, row));");
-                    } line("} catch (RuntimeException e) {"); {
-                        line("log.error(\"Error storing block {} for stream id {}\", row.getBlockId(), row.getId(), e);");
+                line("protected void loadBlocksToOutputStream(Transaction t, ", StreamId,
+                        " streamId, long firstBlock, long numBlocks, OutputStream os) {");
+                {
+                    line("try {");
+                    {
+                        line("BiMap<", StreamValueRow, ", Long> blockRows");
+                        line("        = KeyedStream.of(");
+                        line("        LongStream.rangeClosed(firstBlock, firstBlock + numBlocks).boxed())");
```

Comment:
This should just be `range()` I think (open interval, not closed)

Reply:
How come tests were passing I wonder?

Reply:
Fixed.